WordPress/Classes/ViewRelated/Gutenberg/GutenbergViewController+InformativeDialog.swift

```diff
@@ -0,0 +1,68 @@
+import Foundation
+
+protocol GutenbergFlagsUserDefaultsProtocol {
+    func set(_ value: Bool, forKey defaultName: String)
+    func bool(forKey defaultName: String) -> Bool
+}
+
+extension UserDefaults: GutenbergFlagsUserDefaultsProtocol {
+
+}
+
+/// This extension handles Alert operations.
+extension GutenbergViewController {
+
+    enum Const {
```

Comment:
Not sure about this Const being public, it looks very specific to this extension.
Do you think it's worth it to make it private?

Reply:
I am using this from unit tests so I can't do this private, that but I will give a more specific name

Reply:
Perfect, thank you! 🙏